third_party/protobuf

```diff
@@ -1 +1 @@
-Subproject commit bba446bbf2ac7b0b9923d4eb07d5acd0665a8cf0
+Subproject commit 8c88762e3ef78b81435ecfba973c2e9d3b74f13b
```

Comment:
you'll also need to update this in the sanity test, otherwise sanity test will fail.